Desktop/Regression_Test_Plan_Minor_Release.md

```diff
@@ -212,103 +212,103 @@ TestID | Test Case | Steps to reprouce| Expected Result | Result | Related Comme
-55 :robot: | Allow resharing disabled and Can Share enabled  | The user cannot reshare files | :heavy_check_mark:| tst_sharing |
+52 :robot: | Enforce password protection on public links | Password should be always required | :heavy_check_mark:| tst_sharing | :robot: oC10
+53 | Enforce password protection on public links<br>Try to disable the pwd | Password should be always required | :heavy_check_mark:| tst_sharing |
+54 :robot: | Set default expiration date  on public link | The expiration date should be set by default | :heavy_check_mark:| tst_sharing | :robot: oC10
+55 | Allow resharing disabled and Can Share enabled  | The user cannot reshare files | :heavy_check_mark:| tst_sharing |
 56 | Restrict users to only share with users in their groups | It should appear only the users from the group |:construction:| |
-57 :robot: | Disallow username autocompletion in share dialogs | Full username needs to be entered in order to sharing | :heavy_check_mark:| tst_sharing |
+57 | Disallow username autocompletion in share dialogs | Full username needs to be entered in order to sharing | :heavy_check_mark:| tst_sharing |
 
 ### 11. Selective_Sync
 
-TestID | Test Case | Steps to reprouce| Expected Result | Result | Related Comment
------------- | ------------- | -------------- | ----- | ------ | ------
-1 :robot: | Check that in the setup options after fill in the server and the user you have to choose the server and local folder to sync| 1. Start to setupo oC Desktop 2. Fill in server, user and password 3. The server and local folder to sync is shown|1. On server (you have to choose what do yo want to sync) 2. local folder (the local folder where the data are sync) | :heavy_check_mark: | tst_syncing |
-2a :robot: | Linux+Mac: Check that in the setup options the checkbox is enable to sync all the account by default | 1. Start to setup oC Desktop 2. Fill in server, user and password 3. Click on "Choose what to sync" | At window is opened with the directory tree |:heavy_check_mark:  | tst_syncing |
+TestID | Test Case | Steps to reprouce| Expected Result | Result | Related Comment | Server
+------------ | ------------- | -------------- | ----- | ------ | ------ | -----
+1  :robot: | Check that in the setup options after fill in the server and the user you have to choose the server and local folder to sync| 1. Start to setupo oC Desktop 2. Fill in server, user and password 3. The server and local folder to sync is shown|1. On server (you have to choose what do yo want to sync) 2. local folder (the local folder where the data are sync) | :heavy_check_mark: | tst_syncing | :robot: oC10,<br> :robot: oCIS |
+2a | Linux+Mac: Check that in the setup options the checkbox is enable to sync all the account by default | 1. Start to setup oC Desktop 2. Fill in server, user and password 3. Click on "Choose what to sync" | At window is opened with the directory tree |:heavy_check_mark:  | tst_syncing | :robot: oC10,<br> :robot: oCIS | 
```

Comment:
Currently skipped for oCIS https://github.com/owncloud/client/blob/0fbaf515aa67e3265a2309ed2033061af53f5e6e/test/gui/tst_syncing/test.feature#L59
```suggestion
2a | Linux+Mac: Check that in the setup options the checkbox is enable to sync all the account by default | 1. Start to setup oC Desktop 2. Fill in server, user and password 3. Click on "Choose what to sync" | At window is opened with the directory tree |:heavy_check_mark:  | tst_syncing | :robot: oC10 | 
```